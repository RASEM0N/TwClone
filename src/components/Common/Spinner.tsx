import React from 'react'
import { CircularProgress } from '@material-ui/core'

interface PropsTypes {
    size: number | string
    color?: 'primary' | 'secondary' | 'inherit' | undefined
}

const Spinner: React.FC<PropsTypes> = ({ size, color = undefined }) => {
    return (
        <CircularProgress
            color={color}
            style={{
                width: size,
                height: size,
            }}
        />
    )
}

export default Spinner
