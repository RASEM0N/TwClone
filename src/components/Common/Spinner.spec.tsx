// @ts-ignore
import Spinner from './Spinner'
import Enzyme, { mount, shallow } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'

Enzyme.configure({ adapter: new Adapter() })

describe('PrimaryPage', () => {
    it('just render', () => {
        const component = shallow(<div>121312412</div>)
        expect(component.debug()).toMatchSnapshot()
        console.log(component.debug())
    })
})
