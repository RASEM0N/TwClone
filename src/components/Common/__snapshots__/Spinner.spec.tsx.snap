// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PrimaryPage just render 1`] = `
"<div>
  121312412
</div>"
`;
